File: Chapter 02/C Programs/2_9.c
#include<stdio.h>
void main()
{
int class_held, class_att;
float Percent_att;
char medical_case;
printf("Enter total number of classes held ");
scanf("%d", &class_held);
printf("Enter total number of classes attended ");
scanf("%d", &class_att);
fflush(stdin);
printf("If there is some medical case enter y ");
scanf("%c", &medical_case);

Percent_att = (class_att / (float)class_held)  *100.0;
if(Percent_att>=80)
{ printf( "Not Debarred");  }
else if (medical_case== 'y')
{ printf( "Not Debarred"); }
else
{ printf( "You are debarred"); }

printf("\nAttendance is %f", Percent_att);
     }

File: Chapter 02/Python programs/2_9.py
class_held = int(input("Total number of classes held "))
class_att= int(input("Total number of classes attended "))
medical_case = input("if there is some medical case enter y ")

Percent_att = (class_att / class_held)  *100
if Percent_att>=80 :
    print("Not Debarred")
elif medical_case== 'y':
    print("Not Debarred")
else:
    print("Debarred")
print("Attendance is ", round(Percent_att,2), "%")

